tests/pr_test.go

```diff
@@ -33,13 +33,13 @@ const latestVersion = "8.15"
 // Use existing resource group
 const resourceGroup = "geretain-test-elasticsearch"
 
-// Set up tests to only use supported BYOK regions
+// Restricting due to limited availability of BYOK in certain regions
 const regionSelectionPath = "../common-dev-assets/common-go-assets/icd-region-prefs.yaml"
 
 // Define a struct with fields that match the structure of the YAML data
 const yamlLocation = "../common-dev-assets/common-go-assets/common-permanent-resources.yaml"
 
-var permanentResources map[string]any
+var permanentResources map[string]interface{}
```

Comment:
Redis has `any` https://github.com/terraform-ibm-modules/terraform-ibm-icd-redis/blob/main/tests/pr_test.go#L40

maybe should be changed in Redis as well?